src/utils/canParseUrl.ts

```diff
@@ -0,0 +1,13 @@
+/**
+ * Returns a boolean indicating whether the given URL string
+ * can be parsed into a `URL` instance.
+ * A substitute for `URL.canParse()` for Node.js 18.
+ */
+export function canParseUrl(url: string): boolean {
```

Comment:
We should replace this with `URL.canParse` once Node.js 20 is the minimal version to use with MSW.